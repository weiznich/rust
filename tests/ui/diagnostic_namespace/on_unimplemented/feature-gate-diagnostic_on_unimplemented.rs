// check-fail


#[diagnostic::on_unimplemented(message = "Foo")]
//~^ERROR use of undeclared crate or module `diagnostic
pub trait Bar {
}

fn main() {
}
