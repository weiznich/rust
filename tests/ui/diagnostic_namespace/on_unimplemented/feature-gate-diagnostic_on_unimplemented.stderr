error[E0433]: failed to resolve: use of undeclared crate or module `diagnostic`
  --> $DIR/feature-gate-diagnostic_on_unimplemented.rs:4:3
   |
LL | #[diagnostic::on_unimplemented(message = "Foo")]
   |   ^^^^^^^^^^ use of undeclared crate or module `diagnostic`

error: aborting due to previous error

For more information about this error, try `rustc --explain E0433`.
